tsconfig.shared.json

```diff
@@ -1,10 +1,10 @@
 {
   "compilerOptions": {
-    "target": "es5",
-    "module": "commonjs",
+    "target": "es2017",
```

Comment:
Do I need to bring down to es5? I've taken this over from the master branch, but I'm not sure about the implications.

Reply:
This should be fine. Where we landed was only the CDN bundle (in @turf/turf) needs to be es5 so it can be imported directly into the browser.